index.html

```diff
@@ -454,58 +463,47 @@ <h4>Script Type</h4>
-                  and MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
-                  or <a>Translation</a> <a>Text</a> objects in other languages for for context and quality verification.</p>
-                  <aside class="note">Even though the output of the dubbing workflow is in the same language as those
-                  <a>Text</a> objects with the matching language,
-                  the <a>Text Language Source</a> of those Text objects remains <a>Translation</a>
-                  since they differ from the original language of the dialogue.</aside>
-              </li>
-              <li>
-                <dfn>Audio Description Script</dfn>:
-                <p>When the <code>daptm:scriptType</code> value is <code>AUDIO_DESCRIPTION</code>,
-                  the document represents a <a>transcript</a> of the important visual contents of the
-                  video written to be suitable for use as a <a>script</a>
-                  for generating an audio representation.
-                  It contains text, times, and optional links to audio and mixing instructions
-                  for the purpose of producing an audio description audio track.</p>
-                <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Original</a> <a>Text</a> objects
-                  and MAY contain <a>Translation</a> <a>Text</a> objects in other languages.
-                  If the <a>Target Recording Language</a> is different to the <a>Primary language</a>
-                  then <a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Translation</a> <a>Text</a> objects
-                  in the <a>Target Recording Language</a>.</p>
+
                 <aside class="note">The <a>Text</a> objects in Audio Description <a>Script Events</a> are each expected to have
                   a <a>Text Language Source</a> property of <a>Original</a> if their language is the <a>Primary Language</a>
                   or, when representing in-image text,
                   if they are in the same language as that text.
                   If audio description <a>scripts</a> are translated,
                   their translations would be marked as <a>Translation</a>.</aside>
               </li>
+              <li>
+                <dfn>As-recorded Script</dfn>:
+                <p>When the <code>daptm:scriptType</code> value is <code>asRecorded</code>, 
+                the document represents the actual audio recording.</p>
+                <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Text</a> objects
+                  in the <a>Target Recording Language</a>
+                  and MAY also contain <a>Original</a> <a>Text</a> objects from the <a>Original Language Transcript</a>
+                  or <a>Translation</a> <a>Text</a> objects in other languages for for context and quality verification.</p>
```

Comment:
Since there's no longer an Audio Description script type we need to capture where the mixing instructions come from:

```suggestion
                  or <a>Translation</a> <a>Text</a> objects in other languages for for context and quality verification.
                  They MAY also contain links to audio and mixing instructions for the purpose of producing an audio track incorporating the recordings.</p>
```